.mailmap

```diff
@@ -47,6 +48,7 @@ Andrew Poelstra <asp11@sfu.ca> <apoelstra@wpsoftware.net>
 Anhad Singh <andypythonappdeveloper@gmail.com>
 Antoine Plaskowski <antoine.plaskowski@epitech.eu>
 Anton Löfgren <anton.lofgren@gmail.com> <alofgren@op5.com>
+apiraino <apiraino@users.noreply.github.com> <apiraino@protonmail.com>
```

Comment:
@Noratrieb that's really nice of you :slightly_smiling_face: but to my understanding the thanks page is (at least currently) intended to track contributions in terms of code and reviews - neither of which I do, so I don't know if I fit in

Reply:
I added you because you showed up already

Reply:
> code and reviews - neither of which I do

1. https://github.com/rust-lang/rust/pulls?q=is%3Apr+author%3Aapiraino+is%3Amerged
2. https://github.com/rust-lang/rust/pull/119959

:^)